src/core_modules/capture-core/metaDataStoreLoaders/programs/queries/queryProgramsOutline.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import { query } from '../../IOUtils';
+
+export const queryProgramsOutline = async () => {
```

Comment:
We need the programsOutline because this is the source for deciding which programs to fully reload. The programsOutline contains just a small part of the programsData, like id and version. We will compare the version from programsOutline to the versions in the cache and if they are different we will refetch the whole program.